src/main/java/taxi/controller/StartPageController.java

```diff
@@ -0,0 +1,15 @@
+package taxi.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class StartPageController extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/start_page.jsp").forward(req, resp);
```

Comment:
I guess index.jsp your start page. Either way, if you want to leave it, fix the name (don’t  use snake case here)
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/start-page.jsp").forward(req, resp);
```